### PR TITLE
Add more help info to character commands

### PR DESCRIPTION
**Pull request by xBelladonna**
- Add help info for `avatar` and `proxy` commands  
- Cinnamon now says `Cleared avatar!` when you issue the `avatar` command without parameters
- Fix permanent invite link in `info` embed (link was renewed)  
- Fix a few typos

### CODE REVIEW

**Comment by xBelladonna** [comment]
- Cinnamon now says "Cleared description!" if the `description` command is sent without parameters

File: bot/commands/character.js [hunk 1]
```diff
@@ -44,7 +50,7 @@ Renames \`<character>\``,
     })
     if(args.length == 0){
       if(charactersList.length == 0){
-        return msg.channel.send(utils.errorEmbed(`This server has no characters\nCreate one with \`${guildSettings.prefix}character add <name>\``))
+        return msg.channel.send(utils.errorEmbed(`This server has no characters\nCreate one with \`${guildSettings.prefix || `@${client.user.tag} `}character add <name>\``))
```
Comment (BeeFox-sys) [inline bot/commands/character.js]: Redundant, guildSettings.proxy will never be undefined

File: bot/commands/commands.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ module.exports = {
       embed = new Discord.RichEmbed()
         .setTitle("Commands:")
         .setDescription(data)
-        .setFooter(`You can send \`${prefix}help [command name]\` to get info on a specific command!`)
+        .setFooter(`You can send \`${prefix || `@${client.user.tag} `}help [command name]\` to get info on a specific command!`)
```
Comment (BeeFox-sys) [inline bot/commands/commands.js]: Redundant, guildSettings.proxy will never be undefined

File: bot/commands/commands.js [hunk 3]
```diff
@@ -38,17 +38,17 @@ module.exports = {
     data.push(`**Name:** ${command.name}`);
 
     embed = utils.warnEmbed()
-      .setFooter(`You can send \`${prefix}help [command name]\` to get info on a specific command!`)
+      .setFooter(`You can send \`${prefix || `@${client.user.tag} `}help [command name]\` to get info on a specific command!`)
```
Comment (BeeFox-sys) [inline bot/commands/commands.js]: Redundant, guildSettings.proxy will never be undefined

File: bot/commands/commands.js [hunk 4]
```diff
@@ -38,17 +38,17 @@ module.exports = {
...
     if (command.aliases) embed.addField(`Aliases:`, `${command.aliases.join(', ')}`);
     if (command.description) embed.addField(`Description:`,`${command.description}`);
     if (command.usage) {
 			usage = ""
 			for(var i = 0; i < command.usage.length; i++){
-				usage += `\n${prefix}${command.name}  ${command.usage[i]}`
+				usage += `\n${prefix || `@${client.user.tag} `}${command.name}  ${command.usage[i]}`
```
Comment (BeeFox-sys) [inline bot/commands/commands.js]: Redundant, guildSettings.proxy will never be undefined

File: bot/commands/commands.js [hunk 5]
```diff
@@ -38,17 +38,17 @@ module.exports = {
...
 			}
 			embed.addField(`Usage:`,usage);
 		}
-    if (command.example) embed.addField(`Example:`,`${prefix}${command.name} ${command.example}`);
+    if (command.example) embed.addField(`Example:`,`${prefix || `@${client.user.tag} `}${command.name} ${command.example}`);
```
Comment (BeeFox-sys) [inline bot/commands/commands.js]: Redundant, guildSettings.proxy will never be undefined

File: bot/commands/settings.js [hunk 6]
```diff
@@ -34,7 +34,7 @@ Sets the server's game name`,
 
 				embed = utils.warnEmbed()
 					.setColor('#ffaa00')
-					.addField('Current Prefix:', guildSettings.prefix, true)
+					.addField('Current Prefix:', guildSettings.prefix || "`Unset`", true)
```
Comment (BeeFox-sys) [inline bot/commands/settings.js]: Redundant, guildSettings.proxy will never be undefined

**Review by BeeFox-sys — CHANGES_REQUESTED** [review]
Edits to [commands.js](https://github.com/PlatypodeCode/CinnamonRP/blob/8115112862b92faf1ec30eaa2ea481d06b15ceee/bot/commands/commands.js) and [settings.js](https://github.com/PlatypodeCode/CinnamonRP/blob/8115112862b92faf1ec30eaa2ea481d06b15ceee/bot/commands/settings.js) are redundant, everything else looks amazing though

**Comment by xBelladonna** [comment]
~~Committed evil~~ Forced pushed for Clean History™️

**Comment by xBelladonna** [comment]
Messiest pull request ever sorry ^^;

**Pull request merged**